### DISCUSSION TITLE
Gopro Hero 11 - Geolcation GPS data -address of where video was taken

### DISCUSSION THREAD

**Discussion by Knicks325** [gopro/labs#650] [Q&A]
Just got a gopro and looking to add an overlay of the address of where the video was taken - after some research I was able to overlay with the date and time now I would like to include the address of where the video was taken I have a Gopro Hero 11

**Comment by dnewman-gpsw** [comment]
Use https://gopro.github.io/labs/control/overlays/

Set the camera to 4K 23/25/30fps in 8-bit (only limited modes support overlays.)
Set the font size to 30 (there is a bug with smaller fonts.)
Select:
![image](https://github.com/gopro/labs/assets/10856084/f20e9a01-2073-4bd5-baea-53f5e8939506)

It will show long/latt in the overlay.
